### ISSUE TITLE
Will the train code on NYU dataset be released？

### ISSUE THREAD

**Issue by pijiangdada** [ChanglongJiangGit/A2J-Transformer#9]

**Comment by ChanglongJiangGit** [comment]
Yes, we will publish our training code before September, which includes the training code on Interhand2.6M, RHD, NYU and HANDS 2017. Thanks for your attention!

**Comment by mywordwin** [comment]
> Yes, we will publish our training code before September, which includes the training code on Interhand2.6M, RHD, NYU and HANDS 2017. Thanks for your attention!

Great, thank you for your outstanding work, looking forward to the release of training codes for other datasets, I am waiting for you here!👍👍👍
